charts/tempo-distributed/values.yaml

```diff
@@ -0,0 +1,125 @@
+# -- Overrides the chart's computed fullname
+fullnameOverride: tempo
+# -- Overrides the chart's computed fullname
+tempo:
+  image:
+    # -- Docker image repository
+    repository: grafana/tempo
+    # -- Overrides the image tag whose default is the chart's appVersion
+    tag: latest
```

Comment:
Please make this tag `null`. If this is `null` the app version from the Chart.yaml will be used.